tests/UrlTest.php

```diff
@@ -70,6 +71,16 @@ public function change_frequency_can_be_set()
     }
 
     /** @test */
+    public function alternate_can_be_added()
+    {
+        $alternate = Alternate::create('defaultUrl', 'en');
```

Comment:
The whole `Alternate` value object feels somewhat heavy. Let's make this more simpler. Let the `addAlternate` accept both and `url` and a `locale`. Store those as an array in the `alternates` propery on `Tag`.